VideoApi/Testing.Common/Configuration/Test.cs

```diff
@@ -34,7 +34,7 @@ public Test()
         public List<Conference> TodaysConferences { get; set; }
         public UpdateTaskRequest UpdateTaskRequest { get; set; }
         public Conference YesterdayClosedConference { get; set; }
-        public List<JudgeInHearingResponse> JudgeInHearings { get; set; }
+        public List<ParticipantInHearingResponse> JudgeInHearings { get; set; }
```

Comment:
Should JudgeInHearings be renamed here?

Reply:
This was left as it is used in some test files and I would expect the person who named it did actually only intend this to be a list of judges in hearings, which is why it has been left alone.